userspace/libsinsp/sinsp.cpp

```diff
@@ -374,8 +391,6 @@ void sinsp::init()
 
 	import_ifaddr_list();
 
-	import_user_list();
```

Comment:
Where is the equivalent of this? I don't see any calls to scap_get_user_list() in libsinsp now. Don't you want to use the user list from libscap to initialize the set of users in the usergroup_manager?